test/unit/csv_exporter_test.rb

```diff
@@ -5,6 +5,9 @@ class CsvExporterTest < ActiveSupport::TestCase
     result = CsvExporter.export(Host::Managed, [:id])
     assert_equal "Id\n", result.next
     assert_equal result.count, Host::Managed.count+1
+    assert_difference('CsvExporter.export(Host::Managed, [:id]).count') do
```

Comment:
not directly related to this but I noticed that previously the test may have had incorrect results as there were no hosts in the database.